### PR TITLE
ref(pageFilters): Add a couple missing param types

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: static/app/components/organizations/pageFilters/utils.tsx [hunk 1]
```diff
@@ -71,14 +71,14 @@ export function getStateFromQuery(
  * Useful for extracting page filter properties from the current URL
  * when building another URL.
  */
-export function extractSelectionParameters(query) {
+export function extractSelectionParameters(query: Location['query']) {
```
Comment (priscilawebdev) [inline static/app/components/organizations/pageFilters/utils.tsx:74]: should we add `undefined` here?

Comment (evanpurkhiser) [reply static/app/components/organizations/pageFilters/utils.tsx:74]: afaict we never use it when undefined right?

Comment (evanpurkhiser) [reply static/app/components/organizations/pageFilters/utils.tsx:74]: oh wait, is it failing for that exact reason 😆

Comment (evanpurkhiser) [reply static/app/components/organizations/pageFilters/utils.tsx:74]: I'll take a closer look

Comment (priscilawebdev) [reply static/app/components/organizations/pageFilters/utils.tsx:74]: yes that's why :)

**Review by k-fish — APPROVED** [review]

**Review by leedongwei — APPROVED** [review]

**Pull request merged**